applications/SystemIdentificationApplication/tests/test_system_identification.py

```diff
@@ -18,7 +18,7 @@ def test_DamagedSystem(self):
 
         data = numpy.loadtxt("auxiliary_files/damaged_problem/measured_data.csv", comments="#", usecols=[0,3,4,5,6], delimiter=",")
         ref_data = numpy.loadtxt("auxiliary_files/damaged_problem/measured_data_ref.csv", comments="#", usecols=[0,3,4,5,6], delimiter=",")
-        self.assertMatrixAlmostEqual(data, ref_data, 9)
+        self.assertAlmostEqual(numpy.linalg.norm(data - ref_data), 0.0, 9)
```

Comment:
why not use the numpy comparison routines like I showed in the other PR? Then you can check every value explicitly

Reply:
I needs to get used to that. Good point :)... Now fixed. :)